_posts/2020-02-27-consumption-happiness.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Reducing Consumption while Increasing Happiness"
+date:   2018-11-25 22:40:05
+categories: jekyll update
+tags: book-review
+---
+
+"It's time to stop pointing fingers, finger pointing gets us no-where, Steve!"
+~ Randy Marsh, South Park
+
+![carbon-footprint](/assets/carbon-footprint.png)
+
+In the past year I have [reduced my carbon footprint](https://en.wikipedia.org/wiki/Carbon_footprint#Measuring_carbon_footprints) to about 10% that of an average American without much effort. How?
+
+Before telling other people how to live, I have found it is a good idea to examine my life thoroughly first.
+I know, I know-- where is the fun in that?
+
+There is a lot of fun to be had here! I found how to minimize my environmental impact while also saving time and cash.
+How you may ask? Are you going to try to sell me something? No, this isn't that kind of blog.
+
+The conveniences providing my life with very minor benefits can easily be removed without noticing.
+Before I get into the nuts and bolts of this *need what you are getting into the bolts* let me
+first start off by saying that I don't sweat the small stuff.
+
+I prefer to see what works and then spread the word on major (simple - another word, like achievable) ways to reduce my carbon footprint.  What follows are a few of my major findings
+
+## The list of Things to Work on
+
+Here are the things to think about and ways to think about reducing these area:
+
+  1. **Consumption**.
+
+      a. Food:
+        - If you think of meat as precious yet limited flavoring, plants become a major source of protein, vitamins, and minerals. Cows and other animals are inefficient to grow for the same caloric and nutritional density. Also, cows fart... a lot.
+
+        - Refrigeration takes lots of energy but dried, pickled and sugared fruit and vegetables are all ways to have plant based food that you can eat any time and without needing to use energy to keep it edible.
```

Comment:
I would end the sentence early: `Refrigeration takes lots of energy.` It's a complete thought. Breaking up into shorter sentences flows better.